sunpy/map/tests/test_header_helper.py

```diff
@@ -11,6 +11,9 @@
 from sunpy.map.header_helper import make_heliographic_header
 from sunpy.util.metadata import MetaDict
 
+import re
+
+combined_pattern_1 = (re.escape("This function does not currently support heliocentric coordinates.") + "|" + re.escape("coordinate needs to be a coordinate frame or an SkyCoord instance.") + "|" + re.escape("The coordinate needs an observation time, `obstime`."))
```

Comment:
Does each test raise a different error message?

Reply:
Most do, I believe tox -e py312 returned 70 instances of failed tests.

Reply:
That is not what I asked, the match should be tailored to the extra message that is output from the test and not a combination of several messages.

The goal of the match is to ensure that the test is outputting the correct error.

Reply:
I have made the suggested changes. Thank You.